Formula/libgetdata.rb

```diff
@@ -12,14 +12,14 @@ class Libgetdata < Formula
   end
 
   option "with-fortran", "Build Fortran 77 bindings"
-  option "with-perl", "Build Perl binding"
```

Comment:
I guess that depends. Anyway, just wanted to check. I'm not feeling overly passionate about this option, thus feel free to keep the PR as-is.